docs/adapting-settings.md

```diff
@@ -373,13 +373,30 @@ The padding (in percent) used by the internal provider
 The skin (set) used by the robohash provider.
 See <https://robohash.org/> for more details.
 
-## AVATAR_ROBOHASH_BACKGROUND
+### AVATAR_ROBOHASH_BACKGROUND
 
 **default**: `'bg0'` (transparent background)
 
 The background used by the robohash provider.
 See <https://robohash.org/> for more details.
 
+## Posts configuration
+
+Theses settings allow you to customize the post feature.
+
+### POST_DISCUSSIONS_ENABLED
+
+**default** `False`
+
+Wether or not discussions should be enabled on posts
```

Comment:
s/wether/whether/

Wether is a prime example of a word that will slip past the spell check. It is easily confused with two of its homonyms, whether and weather. Flying fingers find it easy to miss the single letter that separates them. Unless you’re a farmer, you might not even know that wether is either a:

male sheep or ram (the Oxford Dictionary of Etymology traces its roots to Old English, Old High German, Old Norse and Goth)
or a:
castrated ram or billy goat (according to A Word A Day).

😄

Reply:
:joy: